code/modules/projectiles/projectile/bullets.dm

```diff
@@ -23,6 +23,19 @@
 	damage_type = TOX
 	weaken = 5
 
+/obj/item/projectile/bullet/shrapnel
+
+	name = "shrapnel"
+	damage = 55
```

Comment:
I found out that whenless than 15 or 20 shrapnel (3 or 4 shards) are loaded the chances of actually getting hit at 4+ squares of range is smaller than one would expect. That was my original reasoning when making the damage this high. But yeah, its quite spicy, I will reduce it.